base/array.jl

```diff
@@ -2412,6 +2412,68 @@ findall(x::Bool) = x ? [1] : Vector{Int}()
 findall(testf::Function, x::Number) = testf(x) ? [1] : Vector{Int}()
 findall(p::Fix2{typeof(in)}, x::Number) = x in p.x ? [1] : Vector{Int}()
 
+"""
+    findsubseq(A,B)
+
+Return the indices of one or many existing subsequences B in A.
+A and B may be of type AbstractString or AbstractArray.
+If there are none in A, return an empty array.
+
+See also: [`issubseq`](@ref).
+
+# Examples
+```jldoctest
+julia> findsubseq([7, 2, 7, 8, 5, 9, 7, 8], [7, 8])
+2-element Vector{Int64}:
+ 3
+ 7
+
+julia> findsubseq([7, 2, 7, 8, 5, 9, 7, 8], [4, 6])
+Int64[]
+
+julia> findsubseq("Hello","el")
+1-element Vector{Int64}:
+ 2
+```
+
+"""
+function findsubseq(A::AbstractArray, B::AbstractArray)
+    BinA = findall(isequal(B[1]), A)
+    matchFirstIndex = eltype(eachindex(A))[]
+    for i in BinA
+        if length(A[i:end]) < length(B) continue end
+        if A[i:i + length(B) - 1] == B push!(matchFirstIndex, i) end
```

Comment:
This should be a `view` isntead of a slice

Reply:
Done